data/human/human missions.txt

```diff
@@ -7453,3 +7453,1280 @@ mission "Adelita 5"
+			`	Tavish holds up his glass and looks in it, as if deciding if he's willing to drink it or not. "We need to find a ship that frequents this port of call."`
+			`	The bartender taps his overgrown moldy nails on the bar, "Ah yes, me lord. I can probably do that. For a mere 50,000 credits, that is."`
+			`	Tavish flinches at the number and then turns to look at you, "Your call."`
+			label options
+			choice
+				`	(Have Tavish attempt to talk the barkeep into giving you the information.)`
+					goto tav1
+				`	(Have Tavish attempt to talk the barkeep into giving you the information.)`
+					goto tav2
+					to display
+						has "kormak1"
+				`	(Have Kormak intimidate the barkeep into giving you the information.)`
+					goto kormak1
+					to display
+						not "kormak1"
+				`	(Agree to pay 50,000 credits for the information.)`
+					to display
+						not "kormak1"
+					goto payupfront1
+				`	(Agree to pay 100,000 credits for the information.)`
+					to display
+						has "kormak1"
+					goto payupfront2
+			label tav1
+			`	Tavish's eyes sparkle and he turns back to the bartender, raising his glass and taking a deep swig of its unsavory contents.`
+			`	"My good man, you seem like a discerning fellow. Always looking to make a deal with an eye out for profit, am I right?"`
+			`	The bartender looks piqued and slowly nods, "Aye."`
+			`	"You're probably wondering what a pompous overdressed man like me is doing in your quaint out-of-the-way boutique. Well, I was looking for just the sort of enterpising soul who knows a thing or two about making a shilling."`
+			`	Tavish proceeds to describe a long and complex scam utilizing unwitting pirates, corrupt Syndicate trade officials, and fabulously wealthy marks from the paradise world. As he talks, the barkeep leans further and further in, his eyes widening beneath a growing devious grin."`
+			`	Finally, Tavish concludes with, "We're kindred spirits, you and I, and that's why I know this will work. So what do you say? 30,000 credit buy in, plus the information on our first unwitting mark, and you're all set to be rich."`
+			`	"I'm in!" The bartender can't move fast enough, and immediately begins transferring the funds. "Who are you looking for?"`
+			choice
+				`	"We're looking for Hans Fuller of the Broken Record."`
+			action
+				payment 30000
+			`	"Ha, that poor sod," the bartender can't help but chuckle to himself. "Last I was hearing, he was snatched up by the Syndies. By now he's probably on his way back to his hole on Buccaneer Bay. I'd look for 'em around the Persian system."`
+			`	Tavish thanks the man and promises him he'll be seeing his first returns in the near future, and with that the three of you head out of the bar.`
+
+			label story
+			choice
+				`	"Tavish, what's your story?"`
+				`	"Tavish that was amazing! Where did you learn to do that?"`
+				`	"Tell me the truth, Tavish. Are you conning me too?"`
```

Comment:
I'd give a choice to skip this, and cut out at least one of the options here.

Reply:
Skip your ability to ask him what his story is after he talks a dude into paying you for the information you need?

Reply:
Yes. This is great for character development, but at the end of the day not every player cares about player development or is playing a captain who would ask nosy questions about another character.